docs/static/releasenotes.asciidoc

```diff
@@ -59,6 +59,14 @@ This section summarizes the changes in the following releases:
 [[logstash-7-16-1]]
 === Logstash 7.16.1 Release Notes
 
+[[known-issue-7-16-1]]
+==== Known issue
+
+Release 7.16.1 includes updated versions of filters that will not run on JDK 8.
+We are actively patching the build scripts for all filters that are affected. 
```

Comment:
Two affected plugins identified, both patched and released:
 - GeoIP Filter 7.2.7
 - Date Filter 3.1.12

User can upgrade these in the normal way:

~~~
bin/logstash-plugin update logstash-filter-date
bin/logstash-plugin update logstash-filter-geoip
~~~